app/assets/stylesheets/components/tabs.scss

```diff
@@ -3,7 +3,7 @@
 .crayons-tabs {
   // Setup
   --border-width: 3px;
-  --tab-vertical-padding: var(--su-3);
+  --tab-vertical-padding: var(--su-2);
```

Comment:
Same goes for tabs. This could be another PR.

Reply:
Similar to what I mentioned re Modals: these Tabs, as frontend component, are quite new addition that isn't really used anywhere except Notifications (which I've introduced last week) so I'm quite positive this change will not have negative consequences in other places.

Reply:
I realize it's not used yet. I'm just looking for ways to break up the PR. 😃

Reply:
Yea I know. And as I mentioned in one of our calls - big PRs are bad :). But that one line won't really change perception of this PR :D:D:D:D